AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6985 @@
+{
```

Comment:
<p>Amazing job, the new section recaps everything quite well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>